ssl/ech/ech_local.h

```diff
@@ -58,9 +58,16 @@
  * defined in a header file I could find.
  */
 #   define CLIENT_VERSION_LEN 2
-
 #  endif
 
+/*
+ * This is also in ssl_local.h but due to mutual includes
+ * we need it here too, or somewhere used by both
+ */
+# ifndef OPENSSL_CLIENT_MAX_KEY_SHARES
+#  define OPENSSL_CLIENT_MAX_KEY_SHARES 4
+# endif
+
```

Comment:
This cannot be duplicated.
Please move it into internal/ssl.h and include that file from ech_local.h.

Reply:
thanks - wasn't sure where to put it